zcu/zte.py

```diff
@@ -77,6 +80,20 @@ def add_header(payload, signature, payload_type, version):
                                                      0, 0, 0, 0,
                                                      0, 0, 0, 0,
                                                      0, 0, 0, 0)))
+    if signature == b'ZXHN F450(EPON ONU)':
```

Comment:
we shouldn't be adding product-specific code like this, instead the options that write this 'style' should be exposed,

Reply:
> we shouldn't be adding product-specific code like this, instead the options that write this 'style' should be exposed,

The reason is that the device 'F450 (EPON)' export configuration has Little-Endian numbers, and the keygen is like an enhanced 'serial_keygen' method to output the result (key + serialA, iv + serialB)!